checks/31ADCA.yaml

```diff
@@ -0,0 +1,32 @@
+id: "31ADCA"
+name: supported SAPHanaSR-angi version 
+group: OS and package versions
+description: |
+  SAPHanaSR-angi version is supported
+remediation: |
+  ## Abstract
+  Installed SAPHanaSR-angi version must be equal or higher than the recommended value
+  ## Remediation
+  Install or upgrade to a supported SAPHanaSR-angi version
+  ## Reference
+  As it is highly recommended to use the HA/DR provider hook when running SAP workloads in HA environments and the HA/DR provider hook support started with SAPHanaSR-angi version 1.2.5, delivered on installation media of 15SP6 and as update package for 15SP5, the version of the SAPHanaSR-angi package must be at least 1.2.5.
```

Comment:
I think this text should go in the abstract, and only leave the docs reference link in the `References`